LABELING.md

```diff
@@ -0,0 +1,51 @@
+
+# Labeling Guidelines
+
+Now that the screenshots are available, they need to be labeled. The labeling phase operates on couples of comparable screenshots.
+
+## Images marked as compatible - y 
+---
+#### Couples of images that are clearly compatible.
+#### They look the same.
+#### firefox\_chrome\_overlay window should nearly overlap them.
+---
+<p align="center"><img src="labeling_guide/y1.png" width=617 height=357></p>
+<p align="center"><img src="labeling_guide/y2.png" width=617 height=357></p>
+<p align="center"><img src="labeling_guide/y3.png" width=617 height=357></p>
+<p align="center"><img src="labeling_guide/y4.png" width=617 height=357></p>
+<p align="center"><img src="labeling_guide/y5.png" width=617 height=357></p>
+
+## Bounding boxes marked as incompatible - n 
+---
+#### Couples of images which are not compatible
+#### They are different.
+#### Mark the parts which are logically different.
+> Improper loading of images, missing text, different design, different languages are marked incompatible.
```

Comment:
For example, improper loading ...

Add `for example` because these are not the only cases.